CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2719,12 +2719,16 @@ end
 -- Permissions --
 -----------------
 do
-	local permissions;
+	local permissionsLoading, permissions = false;
 	function DeveloperConsole.GetPermissions()
+		while permissionsLoading do wait() end
```

Comment:
seems the only reason this is necessary is due to HttpRbxApiService. Would it be possible just to spawn a thread for this call so we don't need to debounce?